.github/workflows/ci.yaml

```diff
@@ -109,7 +109,7 @@ jobs:
   e2e-tests:
     name: E2E Tests
     runs-on: ubuntu-latest
-    timeout-minutes: 20
+    timeout-minutes: 25
```

Comment:
Had to increase it as tests were getting cancelled while running, [example reference](https://github.com/numaproj/numaplane/actions/runs/13235133722/job/36940124528)

Reply:
good job figuring out the cause for that. If we're anywhere close to that number, how about we increase it even more then? maybe 35? 

I guess separately there is also [this](https://github.com/numaproj/numaplane/blob/main/tests/e2e/common.go#L48) other value in the code itself. Do you mind making both of them 35?

Reply:
I wonder if we should add a comment here as well to help us diagnose the problem if we see this same CI failure again?